packages/cli-kit/src/analytics.ts

```diff
@@ -124,10 +124,10 @@ export function getEnvironmentData(config: Interfaces.Config) {
   return {
     uname: `${platform} ${arch}`,
     env_ci: ciPlatform.isCI,
-    env_ci_platform: ciPlatform.name,
+    env_ci_platform: ciPlatform.name ?? '',
```

Comment:
If a field is undefined/null, I think Monorail ignores it. But this way, I guess they will be saved... Have you checked how it works? Probably not the case, but these changes may break a dashboard if we have queries like `where env_ci_platform IS NOT NULL`.

Reply:
I'll revert to what it was, not worth changing it.